foursight_core/deploy.py

```diff
@@ -64,7 +64,12 @@ def build_config(cls, stage, identity=None, stack_name=None,
         """ Builds the chalice config json file. See: https://aws.github.io/chalice/topics/configfile"""
         # dmichaels/2022-07-22/C4-826:
         # Removed value from the Foursight CloudFormation template; get from GAC/etc at runtime.
-        ignored(stage)
+        # TODO: Is it in fact correct that we are not using rds_name here? The changelog for 1.0.0 doesn't mention it,
+        #       and should be retroactively modified to explain any changes in environment variable use due to the
+        #       1.0.0 change so that users can better adapt to this upgrade. (I noticed that one or more FF
+        #       environments were missing this env variable and was searching all over for clues about why
+        #       that might be.) -kmp 19-Aug-2022
```

Comment:
Oh, yes, sorry about that; RDS_NAME will now be in the GAC (with newer, in PR, 4dn-cloud-infra); I will add that the the CHANGELOG.rst.